sys/include/net/gnrc/pktdump.h

```diff
@@ -28,13 +28,23 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_gnrc_pktdump_conf GNRC PKTDUMP compile configurations
+ * @ingroup net_gnrc_conf
+ * @{
+ */
 /**
  * @brief   Message queue size for the pktdump thread
  */
-#ifndef GNRC_PKTDUMP_MSG_QUEUE_SIZE
-#define GNRC_PKTDUMP_MSG_QUEUE_SIZE     (8U)
+#ifdef CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP
+#define CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE  (1<<CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP)
```

Comment:
Style
```suggestion
#define CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE \
    (1 << CONFIG_GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP)
```

Reply:
This strangely results in error. Hence I did now wrap. Also I think its in 100 char limit.